motion/thruster_interface_auv_cpp/include/thruster_interface_auv_cpp/thruster_interface_auv_ros.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef THRUSTER_INTERFACE_AUV_NODE_HPP
+#define THRUSTER_INTERFACE_AUV_NODE_HPP
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int16_multi_array.hpp"
+#include "vortex_msgs/msg/thruster_forces.hpp"
+#include "thruster_interface_auv_driver.hpp"
```

Comment:
Only use "" for your own project headers.
For headers from external libraries use <>